Docker.md

```diff
@@ -0,0 +1,110 @@
+| /usr/local/tomcat/conf/server.xml | mount/copy of your server.xml | Contains the default server.xml of Tomcat, replace to secure your application |
+| /usr/local/tomcat/conf/catalina.properties | Server properties, contains default framework values | Do not replace this file, use [Environment variables](#Environment-variables) or append to the file, see [Dockerfile](docker/appserver/Tomcat/Dockerfile) for an example |
+
+## Permissions
+
+As this image does not run Tomcat using the root user, file permissions need to be set correctly. By default this is done during startup, to ensure that all files copied to the above locations have the correct permissions. For images with a large number of files in these locations, this can reduce startup performance. It is possible to set the permissions during build and disable the step during startup by adding the following lines at the end of your Dockerfile:
+```
+RUN /setPermissions.sh
+ENV SET_PERMISSIONS_ON_STARTUP=FALSE
+```
+
+Logging
+=======
+
+Generated log files are stored in `/usr/local/tomcat/logs`.
+
+Environment variables
+=====================
+
+Environment variables can be used to set parameters. Environment variables have the highest precedence and override parameters set in .property files supplied by Tomcat, resources and configurations. 
+
+As `org.apache.tomcat.util.digester.PROPERTY_SOURCE=org.apache.tomcat.util.digester.EnvironmentPropertySource` is set in our images, environment variables can also be used to replace parameters in Tomcat configuration files such as server.xml and context.xml.
+
+Considerations
+==============
+
+The images are based on Tomcat, all restrictions and considerations that apply to Tomcat also apply to using the provided images.
+
+## HTTPS and authentication
+
+By default, the image uses the default server.xml of Tomcat which is not configured for inbound HTTPS traffic and user authentication on the administration console. To secure your application, replace server.xml with a secured version matching your requirements.
+
+Note: The GUI will not load data if accessed via HTTP and `dtap.stage!=LOC`, HTTP-based listeners will still process messages.
+
+## Secrets
+
+Special consideration should be taken with secrets. As described on the [Tomcat website](https://cwiki.apache.org/confluence/display/TOMCAT/Password), passwords are stored in plain text. 
+
+We provide the following options to include secrets:
+- Via environment variables as described in the [Tomcat documentation](https://tomcat.apache.org/tomcat-8.5-doc/config/systemprops.html#Property_replacements). By default all environment variables and their values are visible on a number of pages in the console and in the log files, to hide those values `properties.hide` should include the environment variables to hide.
+-
```

Comment:
Not really sure which other options we currently have, #1526 is not yet implemented, and I haven't tried reading from file yet and I'm not sure how to let tomcat read those additional files in a sensible way (would also suffer from the same limitations as environment variables regarding visibility in the console and logging).